src/OpenLoco/src/Objects/Object.h

```diff
@@ -45,6 +45,7 @@ namespace OpenLoco
         scenarioText,
     };
 
+    constexpr size_t kMaxObjectTypes = 34;
```

Comment:
This should be in S5/Limits.h

Reply:
I'm a bit torn over moving it there. I like it here as its just below the object types. I'm not actually sure its even used in the s5 files as a limit its just used when iterating types.